src/sentiment.jl

```diff
@@ -8,6 +8,11 @@ function pad_sequences(l, maxlen=500)
             push!(res, ele)
         end
         return res
+    else
+        throw(1)
+        print("String exceeds maximum length")
```

Comment:
I am afraid that this will throw something like this - `ERROR: 1` and will never execute the print statement even if it executes the else-block.

Try looking for docs (online or offline) for `error` and `throw`. You can easily display error message using those APIs.

Reply:
Here are examples that might help you - [[1]](https://github.com/JuliaText/TextAnalysis.jl/blob/bd4b84dba34554bc2ef14e8554b82522867f28d6/src/stemmer.jl#L18), [[2]](https://github.com/JuliaText/TextAnalysis.jl/blob/bd4b84dba34554bc2ef14e8554b82522867f28d6/src/metadata.jl#L147)

Reply:
@Ayushk4 I have made the changes desired and the function seems to be working fine as when I call it by modifying the maxlen variable it shows the desired Output.
Also I was wondering if XLNet is implemented for Julia as I think it will be a reasonably good addition

Reply:
Yes, XLNet will be a good addition. You can check out Flux.jl and the transformers are in transformers.jl.